index.html

```diff
@@ -18,174 +18,211 @@
         Ut enim ad minim veniam, 
         quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
-    </p>
-    <div class="button">
-      <div class="btn" id="unique">Get my CV</div> 
-      <div id="ic">
-        <a href="#"><img src="img/ic_download.png" alt=""></a>
+      </p>
+      <div class="button">
+       <div class="btn" id="unique">Get my CV</div> 
+        <div id="ic">
+        <a href="#" id="cv-image"><img src="img/ic_download.png" alt=""></a>
+        </div>
       </div>
-    </div>
-  </section>
+    </section>
 
-  <section class="container4">
-    <h1 class="title-about">Languages</h1>
-    <div class="list">
-      <ul>
-        <li>JavaScript(ES5/ES6)</li>
-        <li>Ruby</li>
-        <li>HTML</li>
-        <li>CSS</li>
-      </ul>
-    </div>
-  </section>
+    <section class="container4">
+      <h1 class="title-about">Languages</h1>
+     <div class="list">
+        <ul>
+          <li>JavaScript(ES5/ES6)</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+      </div>
+    </section>
 
-  <section class="container5">
-    <h1 class="title-about">Frameworks</h1>
-    <div class="list">
-      <ul>
-        <li>Bootstrap</li>
-        <li>Ruby on rails</li>
-        <li>RSpec</li>
-        <li>Capibara</li>
-        <li>Selenium</li>
+    <section class="container5">
+      <h1 class="title-about">Frameworks</h1>
+      <div class="list">
+        <ul>
+          <li>Bootstrap</li>
+          <li>Ruby on rails</li>
+          <li>RSpec</li>
+          <li>Capibara</li>
+          <li>Selenium</li>
 
-      </ul>
-    </div>
-  </section>
+        </ul>
+      </div>
+    </section>
+
+    <section class="container6">
+      <h1 class="title-about">Skills</h1>
+      <div class="list">
+        <ul>
+          <li>Codekit</li>
+          <li>Codepen</li>
+          <li>Github</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+       </ul>
+      </div>
+    </section>
+  
+
+  
+    <section class="contact">
+      <div class="corner__geometry">
+         <img src="./images/corner_geometry.svg">
+      </div>
+
+      <h2 class="h2">Get Started</h2>
+      <p class="contact__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+    <form action="https://formspree.io/f/xeqdpkpy" method="post" class="contact__form"> <!-- change made -->
+      <input type="text" required maxlength="30" placeholder="your name" name="name" class="name">
+      <input type="email" required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" placeholder="your email" name="email" class="email">
+      <textarea type="text" maxlength="500" placeholder="Enter your message here..." name="message" id="" class="message"  required> </textarea>
+      <input type="submit" value="Start Collaboration" class="buttons contact__button">
+     </form>
```

Comment:
- [OPTIONAL] I noticed that the font-size of the submit button is too large, so I suggest decreasing the font-size of the button to match your Figma design 💯